intranet/static/js/eighth/statistics.js

```diff
@@ -0,0 +1,79 @@
+/* global $ */
+$(document).ready(function() {
+    if (Object.keys(raw_data).length > 0) {
+        $("#members-chart").show();
+        var canvas = $("#members-chart")[0];
+        var parsed_data = [];
+        $.each(raw_data, function(k, v) {
+            parsed_data.push([Date.parse(k, "YYYY-MM-DD"), v]);
+        });
+        parsed_data = parsed_data.sort(function(a, b) {
+            if (a[0] < b[0]) return -1;
+            if (a[0] > b[0]) return 1;
+            return 0;
+        });
+        var data = {
+            labels: $.map(parsed_data, function(v) { return v[0]; }),
+            datasets: [
+                {
+                    label: "A Block",
+                    backgroundColor: "rgba(151,187,205,0.5)",
+                    data: $.map(parsed_data, function(v) { return "A" in v[1] ? v[1]["A"] : 0; })
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [['A'] is better written in dot notation.](https://www.codacy.com/app/tjcsl/ion/file/2623434281/issues/source?bid=3176160&fileBranchId=3299251#l21)